l3kernel/testfiles/m3intarray001.luatex.tlg

```diff
@@ -22,26 +22,19 @@ This is a coding error.
 LaTeX has been asked to create a new control sequence '\g_testa_intarray' but
 this name has already been used elsewhere.
 The current meaning is:
-  select font cmr10 at 0.00014pt
+  \long macro:->\s__intarray 9 
 Defining \g_testa_intarray on line ...
 ! LaTeX3 Error: Access to an entry beyond an array's bounds.
 For immediate help type H <return>.
  ...                                              
 l. ...  }
 An attempt was made to access or store data at position 0 of the array
 '\g_testa_intarray', but this array has entries at positions from 1 to 12.
-! LaTeX3 Error: Integers larger than 2^{30}-1 cannot be stored in arrays.
-For immediate help type H <return>.
- ...                                              
-l. ...  }
-An attempt was made to store -2000000000 at position 1 in the array
-'\g_testa_intarray'. The largest allowed value -1073741823 will be used
-instead.
 ! LaTeX3 Error: Access to an entry beyond an array's bounds.
 For immediate help type H <return>.
  ...                                              
 l. ...  }
-An attempt was made to access or store data at position 13 of the array
+An attempt was made to access or store data at position 0 of the array
```

Comment:
This change is surprising and suggests that the LuaTeX version is losing track of the index number.